vendor/github.com/containerd/containerd/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.16
 
 require (
 	github.com/Microsoft/go-winio v0.4.17
-	github.com/Microsoft/hcsshim v0.8.21
+	github.com/Microsoft/hcsshim v0.8.23
```

Comment:
I'll open a follow-up PR for updating this one (after https://github.com/moby/moby/pull/42993 is merged, or perhaps I'll update that one)